packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -242,17 +243,28 @@ const generatePermutation = (count: number, user: UsersCurrent|null): Array<numb
 
 
 const ReviewVotingPage = ({classes}: {
-  classes: ClassesType,
+  classes: ClassesType
 }) => {
+  const { LWTooltip, Loading, ReviewVotingExpandedPost, ReviewVoteTableRow, SectionTitle, RecentComments, FrontpageReviewWidget, ContentStyles, Error404 } = Components
+
   const currentUser = useCurrentUser()
   const { captureEvent } = useTracking({eventType: "reviewVotingEvent"})
-  
+  const { params, query } = useLocation()
+  const reviewYear = getReviewYearFromString(params.year)
+  if (!reviewYear) return <Error404 />
+
+  const reviewIsActive = reviewYear === REVIEW_YEAR && getReviewPhase()
+
+  let reviewPhase = getReviewPhase(reviewYear)
+  if (reviewIsActive && query.phase) {
```

Comment:
Probably we want to actually check the value of `query.phase` to make sure it conforms to the `ReviewPhase` type.  (Another use for `TupleSet`)